### ISSUE TITLE
Specifying `config.details` does not affect detail listing.

### ISSUE THREAD

**Issue by samussiah** [RhoInc/safety-shift-plot#17]

**Comment by jwildfire** [comment]
@samussiah is this resolved in v1.3.0?

**Comment by samussiah** [comment]
Nope, no details in v1.3.0.  They'd have to be subject-level and I didn't feel like getting into it.

**Comment by jwildfire** [comment]
Yeah, this is a bit of a mess to clean up. Marking for 2.1.0, but low priority.

**Comment by jwildfire** [comment]
Honestly, solution here might just be to remove the option (instead of fixing implementation)
